python/GangaDirac/Lib/Server/DiracCommands.py

```diff
@@ -11,39 +11,46 @@
 from DIRAC.Interfaces.API.DiracAdmin import DiracAdmin
 dirac = Dirac()
 
-# Write to output pipe
-#/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/
-#reserved_stdout = sys.stdout
-#sys.stdout = sys.stderr
-# def output(object):
-#    reserved_stdout.write("CHECK ME " + str(object))
-#    reserved_stdout.write(pickle.dumps(object))
-
 # Dirac commands
 #/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/
 
-def getJobGroupJobs(jg): output(dirac.selectJobs(jobGroup=jg))
+def getJobGroupJobs(jg):
+    ''' Return jobs in a group'''
```

Comment:
I am surprised this causes problems since string escaping should only happen on the literal level, not after the strings are parsed but if you saw problems then I guess it's just being constructed in a weird way.